cmd/context.go

```diff
@@ -0,0 +1,239 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/CircleCI-Public/circleci-cli/api"
+	"github.com/CircleCI-Public/circleci-cli/client"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+
+	"github.com/CircleCI-Public/circleci-cli/settings"
+	"github.com/spf13/cobra"
+)
+
+func newContextCommand(config *settings.Config) *cobra.Command {
+
+	var cl *client.Client
+
+	initClient := func(cmd *cobra.Command, args []string) {
+		cl = client.NewClient(config.Host, config.Endpoint, config.Token, config.Debug)
+	}
+
+	command := &cobra.Command{
+		Use:   "context",
+		Short: "Contexts provide a mechanism for securing and sharing environment variables across projects. The environment variables are defined as name/value pairs and are injected at runtime.",
+	}
+
+	listCommand := &cobra.Command{
+		Short:  "List contexts",
+		Use:    "list <vcs-type> <org-name>",
+		PreRun: initClient,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return listContexts(cl, args[0], args[1])
+		},
+		Args: cobra.ExactArgs(2),
+	}
+
+	showContextCommand := &cobra.Command{
+		Short:  "Show a context",
+		Use:    "show <vcs-type> <org-name> <context-name>",
+		PreRun: initClient,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return showContext(cl, args[0], args[1], args[2])
+		},
+		Args: cobra.ExactArgs(3),
+	}
+
+	storeCommand := &cobra.Command{
+		Short:  "Store an new secret in the named context. The value is read from stdin.",
+		Use:    "store-secret <vcs-type> <org-name> <context-name> <secret name>",
```

Comment:
Do we call them secrets elsewhere? I thought we normally called them values

Reply:
🤔 

![image](https://user-images.githubusercontent.com/448001/75042347-2c2ab980-54b6-11ea-82b5-2bda21508839.png)

We call them "Environment Variables" in the UI, and "Resources" in the GraphQL schema.

Reply:
FWIW `web-ui-org-settings` renames them "Environment Variables" as well in its shim:

https://github.com/circleci/web-ui-org-settings/blob/5bcdce25669ba7fe8c5bfc91d4035f0f5199a49c/src/graphql/schema.graphql#L23